### PR TITLE
V2.3.0 

### PR DESCRIPTION
**Pull request by GianlucaGuarini**
I would like to publish riot 2.3.0 please let me know if I forgot anything 
@aMarCruz @cognitom @tipiirai @rsbondi

### CODE REVIEW

**Comment by cognitom** [comment]
@GianlucaGuarini nice job! Looks pretty good ;-)

File: release-notes.md [hunk 1]
```diff
@@ -6,6 +6,30 @@ id: release-notes
...
+- [route](https://github.com/riot/route)
+- [core](https://github.com/riot/riot)
+- [cli](https://github.com/riot/cli)
+
+Here the list of our biggest changes:
+
+- Add html5 history router please check [the documentation](/api/route/)
+- Add new rewritten versions of the compiler, template engine and the command line
+- Deprecate `riot.mountTo`
+- Change tag._id to tag._riot_id __you should not use it__
```
Comment (cognitom) [inline release-notes.md]: release notes riot js

It should be wrapped by backquotes:
```
Change `tag._id` to `tag._riot_id` __you should not use it__
```

**Comment by cognitom** [comment]
@GianlucaGuarini can I merge #47 before? It could cause a conflict only on `_config.yml`.

**Comment by GianlucaGuarini** [comment]
> @GianlucaGuarini can I merge #47 before? It could cause a conflict only on _config.yml.

yes merge it 4 me it's ok

> It should be wrapped by backquotes:

yes good feedback I will fix it tonight and release riot 2.3.0

**Comment by cognitom** [comment]
Ok, merged :smile:

File: api/route.md [hunk 2]
```diff
@@ -30,50 +34,186 @@ id = '987987'
...
 
+<span class="tag red">&gt;= v2.3</span>
+
+Riot doesn't `start` its router automatically. DON'T FORGET TO START IT BY YOURSELF. This also means that you can choose your favorite router.
+(Note: before v2.3 Riot started the router automatically. The behavior was changed)
+
 ### riot.route.stop()
 
-Remove the hashchange listeners clearing also the [riot.route](#route) callbacks.
+Stop the all routings. It'll removes the listeners and clear also the callbacks.
```
Comment (MartinMuzatko) [inline api/route.md]: "Stop all the routings" sounds more english to me.

Comment (cognitom) [inline api/route.md]: Oops, exactly. It's my typo. Thanks!

File: guide/index.md [hunk 3]
```diff
@@ -739,6 +751,9 @@ Again, the expression can be just a simple property or a full JavaScript express
 
 The equality operator is `==` and not `===`. For example: `'a string' == true`.
 
+<span class="tag red">important</span>
```
Comment (MartinMuzatko) [inline guide/index.md]: I would move this to a "known issues" page/chapter

File: api/route.md (hunk 2 above)
Comment (cognitom) [inline api/route.md]: Done. https://github.com/riot/route/commit/f2c3f3cf53f44e9d3abc1ec333d9d2d35b33b366

**Comment by MartinMuzatko** [comment]
Great work guys! :)

**Comment by GianlucaGuarini** [comment]
riot 2.3.0 is out I will publish the new doc

**Comment by sylvainpolletvillard** [comment]
Congratulations all, huge step forward indeed :+1: 

Once you checked all the docs for 2.3.0, please push a tag so we can start updating translations as soon as possible.

**Comment by cognitom** [comment]
@sylvainpolletvillard I think we're already close to v2.3.1. It seems better to wait v2.3.1 for translation. I'll let you know again.

**Pull request merged**